gamefixes-steam/976590.py

```diff
@@ -0,0 +1,8 @@
+"""Game fix for Bus Simulator 21 Next Stop"""
+
+from protonfixes import util
+
+
+def main() -> None:
+    # Requires vcrun2022 to launch
+    util.protontricks('vcrun2022')
```

Comment:
[There isn't an upstream issue at all for this title](https://github.com/ValveSoftware/Proton/issues?q=976590), so a fix shouldn't be needed for the Steam version. If there is an issue for it, please report it.

Reply:
Furthermore this game is Deck Verified and [if we're to trust the ProtonDB user reports, there shouldn't be any issues](https://www.protondb.com/app/976590). So I would prefer to keep the fix local to EGS.